### PR TITLE
feat(seo): Add sitemap, robots and description/branding improvements

### PR DESCRIPTION
**Pull request by mezotv**
Adds a sitemap, robots file and tons description/branding improvements.

### CODE REVIEW

File: src/app/layout.tsx [hunk 1]
```diff
@@ -15,8 +15,8 @@ import { Poppins } from 'next/font/google'
 const font = Poppins({ subsets: ['latin'], weight: '400' })
```
Comment (dpaulos6) [inline src/app/layout.tsx:16]:
```ts
function age() {
  var dob = new Date(2003, 5, 17)
  var diff_ms = Date.now() - dob.getTime()
  var age_dt = new Date(diff_ms)
  return Math.abs(age_dt.getUTCFullYear() - 1970)
}
```

File: src/app/layout.tsx [hunk 2]
```diff
@@ -15,8 +15,8 @@ import { Poppins } from 'next/font/google'
...
 
 export const metadata: Metadata = {
-  title: 'dpaulos6',
-  description: "dpaulos6' Personal Portfolio"
+  title: 'Diogo Paulos',
+  description: "I'm Diogo Paulos, and I'm currently 20 years old, living in the beauty of Portugal."
```
Comment (dpaulos6) [inline src/app/layout.tsx:19]:
```suggestion
  description: "I'm Diogo Paulos, and I'm currently {age()} years old, living in the beauty of Portugal."
```

**Review by dpaulos6 — CHANGES_REQUESTED** [review]

File: src/app/layout.tsx (hunk 1 above)
Comment (dpaulos6) [reply src/app/layout.tsx:16]: add this to get my age instead of hard coded.

Comment (mezotv) [reply src/app/layout.tsx:16]: Will do!

File: src/app/layout.tsx [hunk 3]
```diff
@@ -15,8 +15,8 @@ import { Poppins } from 'next/font/google'
 const font = Poppins({ subsets: ['latin'], weight: '400' })
```
Comment (dpaulos6) [inline src/app/layout.tsx:15]:
```suggestion
const font = Poppins({ subsets: ['latin'], weight: '400' })

function age() {
  var dob = new Date(2003, 5, 17)
  var diff_ms = Date.now() - dob.getTime()
  var age_dt = new Date(diff_ms)
  return Math.abs(age_dt.getUTCFullYear() - 1970)
}
```

**Review by dpaulos6 — APPROVED** [review]

**Pull request merged**